ts/components/wallet/WalletHomeHeader.tsx

```diff
@@ -92,50 +70,51 @@ const WalletHomeHeader: React.FC<Props> = (props: Props) => {
     }
   ];
 
-  const openModalBox = async () => {
-    const bottomSheetProps = await bottomSheetContent(
-      <FlatList
-        data={navigationListItems}
-        keyExtractor={item => item.title}
-        renderItem={({ item, index }) => (
-          <>
-            <ButtonDefaultOpacity
-              onPress={() => {
-                closeBS();
-                item.onPress();
-              }}
-              style={styles.container}
-              onPressWithGestureHandler={true}
-            >
-              <View style={styles.flexColumn}>
-                <View style={styles.row}>
-                  <View style={IOStyles.flex}>
-                    <H3 color={"bluegreyDark"} weight={"SemiBold"}>
-                      {item.title}
-                    </H3>
-                    <H5 color={"bluegrey"} weight={"Regular"}>
-                      {item.subtitle}
-                    </H5>
-                  </View>
-                  <IconFont name={"io-right"} color={IOColors.blue} size={24} />
+  const { present, dismiss } = useIOBottomSheet(
+    <FlatList
+      data={navigationListItems}
+      keyExtractor={item => item.title}
+      renderItem={({ item, index }) => (
+        <>
+          <ButtonDefaultOpacity
+            onPress={() => {
+              closeBS();
+              item.onPress();
+            }}
+            style={styles.container}
+            onPressWithGestureHandler={true}
+          >
+            <View style={styles.flexColumn}>
+              <View style={styles.row}>
+                <View style={IOStyles.flex}>
+                  <H3 color={"bluegreyDark"} weight={"SemiBold"}>
+                    {item.title}
+                  </H3>
+                  <H5 color={"bluegrey"} weight={"Regular"}>
+                    {item.subtitle}
+                  </H5>
                 </View>
+                <IconFont name={"io-right"} color={IOColors.blue} size={24} />
               </View>
-            </ButtonDefaultOpacity>
+            </View>
+          </ButtonDefaultOpacity>
+
+          {index !== navigationListItems.length - 1 && (
+            <ItemSeparatorComponent noPadded />
+          )}
+        </>
+      )}
+    />,
+    I18n.t("global.buttons.add"),
+    315
+  );
 
-            {index !== navigationListItems.length - 1 && (
-              <ItemSeparatorComponent noPadded />
-            )}
-          </>
-        )}
-      />,
-      I18n.t("global.buttons.add"),
-      315,
-      closeBS
-    );
+  const closeBS = () => {
+    dismiss();
```

Comment:
Is this needed to wrap a single instruction?

We may use disimiss call directly inside this code block:
```javascript
            <ButtonDefaultOpacity
              onPress={() => {
                dismiss();
                item.onPress();
              }}
              style={styles.container}
              onPressWithGestureHandler={true}
            >
```

what do you think?

Reply:
sure! thx

https://github.com/pagopa/io-app/pull/2614/commits/2ca2f86c3926ecf4602d6bc2061355b3700d2ba4